src/EventSubscriber/RngRequestSubscriber.php

```diff
@@ -54,10 +54,10 @@ public function __construct(EntityTypeManagerInterface $entity_type_manager, Rng
   /**
    * Run RNG rules for entity operations which occurred during this request.
    *
-   * @param \Symfony\Component\HttpKernel\Event\PostResponseEvent $event
+   * @param \Symfony\Component\HttpKernel\Event\TerminateEvent $event
    *   The event to process.
    */
-  public function onKernelTerminate(PostResponseEvent $event) {
+  public function onKernelTerminate(TerminateEvent $event) {
```

Comment:
Please file separate  issue for that, also it needs link to change record where the event was renamed